modify-output-of-many-types.ipynb

```diff
@@ -57,15 +57,22 @@
    "execution_count": 3,
```

Comment:
<p>Test this out</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/smith-kyle/notebook-diff-tests/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>